src/components/user/services/user.service.v2.integration.test.js

```diff
@@ -0,0 +1,163 @@
+		await testObjects.createTestAccount(credentials, 'local', adminUser);
+		const token = await testObjects.generateJWT(credentials);
+		return token;
+	};
+
+	const getPermissionToken = async (schoolId, permissions = []) => {
+		const currentUserRole = `currentUserPermission`;
+		await testObjects.createTestRole({
+			name: currentUserRole,
+			permissions,
+		});
+		const currentUser = await testObjects.createTestUser({
+			firstName: 'deleteUser',
+			roles: [currentUserRole],
+			schoolId,
+		});
+		const credentials = { username: currentUser.email, password: `${Date.now()}` };
+		await testObjects.createTestAccount(credentials, 'local', currentUser);
+		const token = await testObjects.generateJWT(credentials);
+		return token;
+	};
+
+	describe('API tests', () => {
+		it('When an authorized user deletes a student and returns success', async () => {
+			const { _id: schoolId } = await testObjects.createTestSchool();
+			const user = await testObjects.createTestUser({ roles: ['student'], schoolId });
+			const token = await getAdminToken(schoolId);
+			const request = chai
+				.request(app)
+				.delete(`/users/v2/admin/student/${user._id.toString()}`)
+				.query({ userId: user._id.toString() })
+				.set('Accept', 'application/json')
+				.set('Authorization', token)
+				.set('Content-type', 'application/json');
+			const response = await request.send();
+			expect(response.status).to.equal(200);
+			expect(response.body.userId).to.deep.equal(user._id.toString());
+		});
+
+		it('Fails when not authorized user deletes a student', async () => {
+			const { _id: schoolId } = await testObjects.createTestSchool();
+			// const params = await testObjects.generateRequestParamsFromUser(admin);
+			const teacher = await testObjects.createTestUser({ roles: ['teacher'], schoolId });
+			const user = await testObjects.createTestUser({ roles: ['student'], schoolId });
+			const token = await testObjects.generateJWTFromUser(teacher);
+			const request = chai
+				.request(app)
+				.delete(`/users/v2/admin/student/${user._id.toString()}`)
+				.query({ userId: user._id.toString() })
+				.set('Accept', 'application/json')
+				.set('Authorization', token)
+				.set('Content-type', 'application/json');
+			const response = await request.send();
+			expect(response.status).to.equal(403);
+		});
+
+		it('Returns error is user not found', async () => {
+			const { _id: schoolId } = await testObjects.createTestSchool();
+			const notFoundId = ObjectId();
+			const token = await getAdminToken(schoolId);
+			const request = chai
+				.request(app)
+				.delete(`/users/v2/admin/student/${notFoundId.toString()}`)
+				.query({ userId: notFoundId.toString() })
+				.set('Accept', 'application/json')
+				.set('Authorization', token)
+				.set('Content-type', 'application/json');
+			const response = await request.send();
+			expect(response.status).to.equal(404);
+		});
+
+		it('Fails when user from different deletes a student', async () => {
```

Comment:
missing a word here?
WHEN a user from a different school tries to delete a student, THEN a 404 error is thrown

Dont be afraid of long test titles, they will help you when looking for the cause of a failed test.